example-apps/my-nest-project/src/users/user-history.entity.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  History,
+  HistoryActionKind,
+  HistoryColumn,
+} from '@kittgen/nestjs-typeorm-history';
+import { Column, Entity } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserHistory extends User implements History {
```

Comment:
this is a bit dangerous. Especially because of the id field UserHistory should contain User

Reply:
as discussed: make composition, add 2nd strategy for supporting JSON blobs